libraries/aesthetic/src/main/java/com/afollestad/aesthetic/Aesthetic.java

```diff
@@ -69,22 +71,33 @@
   private static Aesthetic instance;
 
   private final ArrayMap<String, Integer> lastActivityThemes;
+  private final Context appContext;
 
   private CompositeDisposable subs;
-  private Context context;
+  private WeakReference<Context> weakRefContext;
```

Comment:
I don't really see the purpose of this weak reference. If the context is cleared in onPause(), there's no need for a weak reference.

Reply:
Somehow LeakCanary detect it leak, maybe this condition doesn't work as expected in some specific scenarios:
```java
if (activity.isFinishing()) {
  if (context == activity) {
    context = null;
  }
}
```
Wrap it to weak reference object for safe, since holding activity instance as in static var is not a good design.